core/src/main/java/com/onewelcome/core/manager/PreferencesManager.kt

```diff
@@ -0,0 +1,6 @@
+package com.onewelcome.core.manager
+
+interface PreferencesManager {
+  suspend fun isSdkAutoInitializationEnabled(): Boolean
```

Comment:
I think this one doesn't have to be `suspend`

Reply:
I've added 

```  override suspend fun isSdkAutoInitializationEnabled(): Boolean {
    return dataStore.isSdkAutoInitializationEnabled().first()
  }
 ```
 
 .first() requires suspend function. I wanted to return `Boolean` instead of `Flow<Boolean>` so that's why the `first()` here